go/bin/spec

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+bin/lint
+
+# runs go test on all subdirectories exluding the vendor folder
+# might cause issues with windows users though
```

Comment:
http://gnuwin32.sourceforge.net/packages/grep.htm
How about updating the comment to: `(windows users need to 'grep' installed)`